cafy_pytest/plugin.py

```diff
@@ -1324,7 +1367,8 @@ def pytest_runtest_logreport(self, report):
             if hasattr(CafyLog,"model_tracker_dict"):
                 self.model_coverage_report[testcase_name]=CafyLog.model_tracker_dict
                 CafyLog.model_tracker_dict={}
-
+            if hasattr(CafyLog,"collection") and CafyLog.collection:
+               self.collection_manager.disconnect()
```

Comment:
if setup_module fails, then teardown_module doesnt get called, in that cse this collection_manager.disconnect() may not be guaranteed.